From: Jiri Slaby <jslaby@suse.cz>
Date: Wed, 4 Apr 2018 15:29:47 +0200
Subject: Revert "tools: fix cross-compile var clobbering"
Patch-mainline: not yet, reported on 2018/04/01
References: build fix

This partially reverts commit 7ed1c1901fe52e6c5828deb155920b44b0adabb1,
as it breaks build of acpica (acpidump):
 ld: unrecognized option '-D_LINUX'

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 tools/power/acpi/Makefile.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/power/acpi/Makefile.config b/tools/power/acpi/Makefile.config
index 2cccbba64418..c7bcf482da55 100644
--- a/tools/power/acpi/Makefile.config
+++ b/tools/power/acpi/Makefile.config
@@ -56,6 +56,7 @@ INSTALL_SCRIPT = ${INSTALL_PROGRAM}
 # to compile vs uClibc, that can be done here as well.
 CROSS = #/usr/i386-linux-uclibc/usr/bin/i386-uclibc-
 CROSS_COMPILE ?= $(CROSS)
+LD = $(CROSS_COMPILE)gcc
 HOSTCC = gcc
 
 # check if compiler option is supported
-- 
2.16.3

